lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -907,6 +907,7 @@ sub upgrade {
 
     my $settings = $request->{_wire}->get( 'setup_settings' );
     my $auth_db = ($settings and $settings->{auth_db}) // 'postgres';
+    $database->{connect_data}->{options} = "-c search_path=$database->{schema},public";
```

Comment:
Could you add a comment here and in `sub login()` to cross reference these and make sure we keep them in sync?